e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -0,0 +1,200 @@
+import pathlib
+import subprocess
+
+import pytest
+
+from determined.cli import ntsc
+from determined.common import context, util
+from determined.common.api import bindings
+from tests import api_utils
+from tests import config as conf
+
+
+@pytest.mark.e2e_cpu
+def test_create_generic_task() -> None:
+    """
+    Start a simple task with a context directory called from the task CLI
+    """
+    command = [
+        "det",
+        "-m",
+        conf.make_master_url(),
+        "task",
+        "create",
+        conf.fixtures_path("generic_task/test_config.yaml"),
+        "--context",
+        conf.fixtures_path("generic_task"),
+    ]
+
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
```

Comment:
is it guaranteed that `det task create` will fail if the task crashes?

That's not guaranteed for `det cmd run`, where the exit code of `det cmd run` reflects if there was a failure in the CLI itself, not whether or not there was a failure in the commnad that ran.

Reply:
(question applies throughout)

Reply:
`det task create` will only fail if the CLI returns an error. Tasks that crash or error out will have their task state be ERROR. The same with `det task fork`.

Reply:
ok, that's good, because it's consistent with how other CLI commands run in our system.

Should this test then make sure that the task completed successfully?  If it was properly created, it should not fail, right?

Reply:
Tests added to check task completion and failure